src/scss/grommet-core/_objects.form-field.scss

```diff
@@ -78,13 +78,39 @@ $form-vertical-padding: quarter($inuit-base-spacing-unit);
 
     > select {
       display: block;
-      background-position: center right $form-horizontal-padding;
+      // TODO figure out how to make this consistent with intuit and not have magic numbers
+      background-position: center right #{$form-horizontal-padding - 6};
```

Comment:
You can use:

background-position: center right $form-horizontal-padding - quarter($inuit-base-spacing-unit);